elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -195,7 +196,190 @@ class PfeNavigation extends PFElement {
         });
       });
     }
-  }
+
+    // Last link visually
```

Comment:
This is a lot of commented out code?

Reply:
@wesruv thanks for catching that, that was from my original testing and working through the feature. Got that stale code removed.